felix/rules/endpoints.go

```diff
@@ -21,6 +21,7 @@ import (
 	"strings"
 
 	v3 "github.com/projectcalico/api/pkg/apis/projectcalico/v3"
+	"github.com/sirupsen/logrus"
 	log "github.com/sirupsen/logrus"
```

Comment:
Since we are using logrus, can we replace the other calls "log", so that we don't import
 `"github.com/sirupsen/logrus"`
 and
`log "github.com/sirupsen/logrus"`